website/tutorial/basic/02-getting-started.mdx

```diff
@@ -0,0 +1,325 @@
+export const typeDefinitions = /* GraphQL */ `
+  type Query {
+    info: String!
+  }
+`
+
+const resolvers = {
+  Query: {
+    info: () => 'Test',
+  },
+}
+```
+
+</Instruction>
+
+As you can see the resolver map follows the same structure as the type definitions.
+
+The typename (`Query`) is an object in which the fields (`info`) are functions that return the corresponding data of that field.
+
+<Instruction>
+
+Now you still need to glue together the type definitions and the resolver map.
+
+```typescript
+import { makeExecutableSchema } from '@graphql-tools/schema'
+
+export const typeDefinitions = /* GraphQL */ `
+  type Query {
+    info: String!
+  }
+`
+
+export const resolvers = {
+  Query: {
+    info: () => 'Test',
+  },
+}
+
+export const schema = makeExecutableSchema({
+  resolvers: [resolvers],
+  typeDefs: [typeDefinitions],
+})
+```
+
+</Instruction>
+
+In the code-snippet above, you've created or used the following variables:
+
+- `typeDefinitions` - this is your GraphQL schema definition. You've created a `Query` type that exposes a field called `info`, of type `String`. You can import it directly from `.graphql` file thanks to `graphql-import-node`.
+- `resolvers` - the resolver functions are part of the GraphQL schema, and they are the actual implementation (code / logic) of the GraphQL schema.
+- `schema` - a combination of the GraphQL SDL and the resolvers. `makeExecutableSchema` function is in charge of gluing them together into an executable schema we can later use.
+
+Now that you have a GraphQL schema, you can use that to fetch data using a GraphQL `query` operation!
+
+### Query the GraphQL schema
+
+As explained before, the GraphQL schema is only your contract, and it exposes the set of all types and capabilities that your API layer can do.
+
+To use your GraphQL schema and consume data from it, you will need to write an executable GraphQL `query` definition.
+
+Based on the schema your created before, you can use the following query:
+
+```graphql
+query {
+  info
+}
+```
+
+So you don't have to get into all the complexity of running GraphQL server - you can use this query and run it against your GraphQL schema and get an immediate result.
+
+To query our local schema, even without any fancy GraphQL client or even a GraphQL server, you can use GraphQL's `execute` function to just run the schema with the query.
+
+<Instruction>
+
+Update the code in `src/main.ts` to contain the following snippet:
+
+```ts
+import { execute, parse } from 'graphql'
+import { schema } from './schema'
+
+async function main() {
+  const myQuery = parse(/* GraphQL */ `
+    query {
+      info
+    }
+  `)
+
+  const result = await execute({
+    schema,
+    document: myQuery,
+  })
+
+  console.log(result)
+}
+
+main()
+```
+
+</Instruction>
+
+Now, try to run our project again (either with `npm run dev` or `npm run start`), you should see in the output log the following:
+
+```
+{ data: [Object: null prototype] { info: 'Test' } }
```

Comment:
Should we explain the [Object: null prototype] ?

Reply:
We can omit it, but it might confuse people as the console.log will then be different :thinking:

I am not 100% sure what the actual benefit of this it.. Make sure that the object never inherits stuff?

Reply:
added a quick explanation